src/cpp/shared/sound/cd_music.cpp

```diff
@@ -100,9 +102,9 @@ void bass_play_track(int trknum) {
     return; //this track is already being played
 
   if (!useOpera && trknum >= 4 && trknum <= 9)
-    sprintf(trkbuf, "%s/sw/homm2_%02d.ogg", trkpath, trknum);
+    snprintf(trkbuf, sizeof(trkbuf), "%s/sw/homm2_%02d.ogg", trkpath, trknum);
```

Comment:
Thanks for the small patch. As a small FYI: the reason why I haven't been super-strict about making people use snprintf over sprintf is that it's a bit of a lost cause; there's hundreds of uses throughout the code, usually with fixed-size buffers.